gradle/verification-metadata.xml

```diff
@@ -0,0 +1,1809 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<verification-metadata xmlns="https://schema.gradle.org/dependency-verification" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="https://schema.gradle.org/dependency-verification https://schema.gradle.org/dependency-verification/dependency-verification-1.3.xsd">
+   <configuration>
+      <verify-metadata>true</verify-metadata>
+      <verify-signatures>false</verify-signatures>
+      <trusted-artifacts>
+         <trust file=".*-javadoc[.]jar" regex="true"/>
+         <trust file=".*-sources[.]jar" regex="true"/>
+         <trust file="gradle-[0-9.]+-src.zip" regex="true"/>
+         <trust file="groovy-[a-z]*-?[0-9.]+.pom" regex="true"/>
+      </trusted-artifacts>
+   </configuration>
+   <components>
+      <component group="args4j" name="args4j" version="2.33">
+         <artifact name="args4j-2.33.jar">
+            <sha256 value="91ddeaba0b24adce72291c618c00bbdce1c884755f6c4dba9c5c46e871c69ed6" origin="Generated by Gradle"/>
```

Comment:
Is this the hashes of all the dependencies we pull in?

Reply:
yeah. It's necessary for how I build using nixos currently (ref. https://github.com/raphiz/buildGradleApplication)
It can be mostly auto-generated by using gradle: `./gradlew --refresh-dependencies --write-verification-metadata sha256 build`